static/src/javascripts/projects/common/modules/commercial/badges.js

```diff
@@ -28,12 +28,12 @@ define([
     var badgesConfig = {
             sponsoredfeatures: {
                 count:      0,
-                header:     config.switches.newCommercialContent ? 'Supported by:' : 'Sponsored by:',
+                header:     'Supported by:',
                 namePrefix: 'sp'
             },
             'advertisement-features': {
                 count:      0,
-                header:     config.switches.newCommercialContent ? 'Paid for by' : 'Brought to you by:',
+                header:     'Paid for by:',
```

Comment:
There's an email somewhere, but nope the colon stays for all other cases but paid for content.